tests/integration/components/paper-item-test.js

```diff
@@ -0,0 +1,182 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+import wait from 'ember-test-helpers/wait';
+
+const { set, get } = Ember;
+
+moduleForComponent('paper-item', 'Integration | Component | paper item', {
+  integration: true
+});
+
+test('single action checkboxes should react to checkbox clicks', function(assert) {
+  assert.expect(1);
+  set(this, 'checkboxEnabled', false);
+  this.render(hbs`
+    {{#paper-list}}
+      {{#paper-item}}
+        <p>Checkbox 1</p>
+        {{paper-checkbox class="md-secondary" value=checkboxEnabled onChange=(action (mut checkboxEnabled))}}
+      {{/paper-item}}
+    {{/paper-list}}
+  `);
+  return wait().then(() => {
```

Comment:
@miguelcobain previous usage of `didInsertElement` triggered an Ember double modify deprecation. And it is distinct approach as it would only run once event if we would add a new component under paper-item at some point all the triggers wouldn't run simply. The `init` of paper-item is also not the best place as it would require registering an observer to run `setupProxiedComponent`. Then `setupProxiedComponent` would have to have `scheduleOnce` to avoid double invocation. In this situation the tests wouldn't run as `wait` is not so clever to wait for 2 serial async functions.